Revise 0315 - Add RPC conflict management

Akihiro-Miyazaki opened the pull request:
## Introduction
This is an update of SDL-0315.

## Proposed solution
1. For meeting the json implemented example, change the word from "InterruptManagerConfig" to "rpc_priority_table".
2. Modify to "InterruptManager reads rpc_priority_table from the policy table to construct the two tables mentioned above based on their settings." according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-732988661).
3. Add the json example for App priority table, HMI status priority table and policy table according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-733262330).
4. Add the UI.SubtleAleart to the Table 1, Table 2 and json examples according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-732988661).
5. Remove "during the startup of SDL Core" from 1.Loading of configuration table.
6. Change the image files of Figure 2 and Figure 3 for modify the sentence "Load rpc_priority_table from policy table".
7. Change the word from "RPC priority table and App priority table" to "RPC priority table, App priority table and HMI status priority table".
8. Word unification:
     - RPC Priority => RPC priority
     - App Priority => App priority
     - HMI Status => HMI status priority

## Impact on existing code
1. Change the word from "InterruptManagerConfig" to "rpc_priority_table".
2. Change the word from "RPC priority table and App priority table" to "RPC priority table, App priority table and HMI status priority table".

Akihiro-Miyazaki commented:
Hi, @jordynmackool -san,
Thank you for your cooperation as always.
We finished to update the proposal documentation and image files.
The modifications are almost according to [this comments](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-733766674), also some words are changed.
We made a commit them.
Could you please confirm them.
Thank you.

pelsila commented:
Hello @Akihiro-Miyazaki.  It looks like you may have made more changes than the Steering Committee requested in [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-733766674).  Please revise the PR to only include the Steering Committee requested changes for 1, 2, and 3.  

If you have changes that you would like to include, please comment on the issue when it is under review.  Please advise when this is ready and I will re-review.  Thank you

Akihiro-Miyazaki commented:
Hello, @pelsila -san,
Thank you for your reply.
I assume these changes are caused by [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-733766674).

I think it is necessary to unify the word.
No.1 / 6 / 8 in Proposed solution and No.1 in Impact on exiting code are to unify the word.

5 in Proposed solution
No.5 change is same as No.2.
In No.2, "during the SDL Core startup" is wrong.
Since the same word is described in No.5, then I removed this word.

7 in Proposed solution and 2 in Impact on existing code
These changes are also caused by [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-733262330).

However, I forgot to describe to remove the image file (Table4_Priority_result_of_Table3.png). 
It was used at the first proposal documentation.
When I modify the priority table from appHMIType to app, I removed from this proposal documentation.
But this image file is stored in the assets.
Therefore, I removed it from the assets.

If you have any question, please ask to me.
Thank you.
